src/web/mjs/connectors/HiperCool.mjs

```diff
@@ -0,0 +1,73 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class HiperCool extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'hipercool';
+        super.label = 'Hiper Cool';
+        this.tags = [ 'manga', 'portugese', 'hentai' ];
+        this.url = 'https://hiper.cool/';
+    }
+
+    async _getMangas() {
+        let page = 1;
+        let morePages = true;
+
+        let request, data;
+
+        let mangas = [];
+        let uniq = new Set();
+
+        while ( morePages ) {
+            try {
+                request = new Request(new URL('/home/' + page, this.url), this.requestOptions);
+                data = await this.fetchDOM(request, '.news-list > li');
+                page++;
+
+                for (let manga of data) {
+                    if ( !uniq.has(manga.querySelector('a.news-thumb').pathname) ) {
+                        uniq.add(manga.querySelector('a.news-thumb').pathname);
```

Comment:
I think duplicates will already be removed in the base class, so this would be redundant

Reply:
Looks you're right. Didn't know that.